src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -126,7 +126,8 @@ public Integer convert(String value) {
     @Option(names = "-o", description = "Save validation report model to a file.  KB.ttl would result in KB-report.txt.")
     private boolean outputToFile;
 
-    @Option(names = "-t", description = "Specify the number of threads to use during validation", paramLabel = "num")
+    @Option(names = "-t", description = "Specify the number of threads to use during validation. As the threaded validator" +
+            " doesn't currently support adding a progress monitor, this disables the --pm option.", paramLabel = "num")
```

Comment:
Nice.  I think this should probably go in the README also, but I don't object to leaving it out as we intend to add multi-threaded progress monitoring "soon."

Reply:
I would argue that it IS in the README. Copy+paste!

Reply:
Oh my, how did I miss that?  I was looking for it in `--pm` in the README for some reason.  My bad.